### PR TITLE
Nat-Jules-Saturday

### PR DESCRIPTION
**Pull request by nataliegarate**
### Assignee Tasks

* [ ] added unit tests (or none needed)
* [ ] written relevant docs (or none needed)
* [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

_Your PR Notes Here_

### CODE REVIEW

File: client/components/checkout.js [hunk 1]
```diff
@@ -10,6 +11,8 @@ class CheckOut extends React.Component {
     this.props.getCartThunk()
   }
   render() {
+    console.log(this.props)
```
Comment (EvaE-P) [inline client/components/checkout.js]: could we take this console.log out?

File: client/components/auth-form.js [hunk 2]
```diff
@@ -11,23 +11,6 @@ const AuthForm = props => {
 
   return (
```
Comment (EvaE-P) [inline client/components/auth-form.js]: when was this form created?

File: client/components/oneCupcake.js [hunk 3]
```diff
@@ -35,36 +33,18 @@ class SingleCupcake extends React.Component {
       quantity: 0,
       id: cupcake.id
```
Comment (EvaE-P) [inline client/components/oneCupcake.js]: this looks different to me now - i remember it as "this.props.match.params.id " any spcecific reason for that change?

Comment (nataliegarate) [reply client/components/oneCupcake.js]: on my branch it says:
  this.state = {
      quantity: 0,
      cupcakeId: this.props.match.params.id
    }

Comment (nataliegarate) [reply client/components/oneCupcake.js]: Maybe this was just showing something deleted?

File: client/components/auth-form.js (hunk 2 above)
Comment (nataliegarate) [reply client/components/auth-form.js]: This form initially came with the Boiler Plate. Currently it just has email and password. We deleted any other areas of input

File: client/components/oneCupcake.js (hunk 3 above)
Comment (nataliegarate) [reply client/components/oneCupcake.js]: https://github.com/fantastic-four-grace-shopper/grace-shopper/blob/Nat-Jules-Saturday/client/components/oneCupcake.js

File: client/components/auth-form.js (hunk 2 above)
Comment (julianaalbano) [reply client/components/auth-form.js]: hi @EvaE-P! this was provided via the boilermaker repo. This is the sign up component. Natalie and I just deleted the "first name" and "last name" input fields that we are not using.

File: client/components/oneCupcake.js (hunk 3 above)
Comment (nataliegarate) [reply client/components/oneCupcake.js]: This is master and the previous was listed above  https://github.com/fantastic-four-grace-shopper/grace-shopper/blob/master/client/components/oneCupcake.js

**Pull request closed without merging**